src/ui/component/dayPicker/dayPicker.tsx

```diff
@@ -0,0 +1,67 @@
+import { type FC, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { ActiveModifiers } from 'react-day-picker'
+import { DayPicker as RDayPicker } from 'react-day-picker'
+import { enUS, fr, de } from 'date-fns/locale'
+import 'react-day-picker/dist/style.css'
+import './dayPicker.scss'
+
+const localeMap = new Map([
+  ['en', enUS],
+  ['fr', fr],
+  ['de', de]
+])
+
+type DayPickerProps = {
+  onSelect: (
+    day?: Date,
+    // @ts-expect-error built-in ract-day-picker handler signature has optional first parameter
```

Comment:
Why this ts rule comment?

Reply:
If we don't provide error comment, TS won't compiled because of [error TS1016](https://typescript.tv/errors/#TS1016). We can find a solution to avoid it and address your [comment](https://github.com/okp4/dataverse-portal/pull/454#discussion_r1317363263) related to _dependency inversion principle_.